background.js

```diff
@@ -0,0 +1,4 @@
+import "/js/allTenants.js";
```

Comment:
use ES6 module syntax to import multiple script files (v3 manifest requirement).